### PR TITLE
Fixes HTML  issues

### PR DESCRIPTION
**Pull request by crablab**
Passes HTML proofer tests to make Circle CI happy :blush:

### CODE REVIEW

**Comment by crablab** [comment]
Could you describe the issues with Circle CI a bit more please? :)

File: circle.yml [hunk 1]
```diff
@@ -29,4 +29,4 @@ jobs:
       
       - run:
           name: Test
-          command: bundle exec htmlproofer ./_site --check-html --disable-external
+          command: bundle exec htmlproofer ./_site --check-html --disable-external --external_only
```
[the remark this answers is not in the document]
Comment (crablab) [reply circle.yml]: They are mutually exclusive! It actually fixes a weird bug which causes HTML proofer to not resolve links that are valid. It's required to pass the tests - if you don't have the flag your tests will fail due to an unsolvable issue (which actually isn't an issue).

**Comment by chebyrash** [comment]
Yeah I don't think my review counts towards anything 😄

**Comment by crablab** [comment]
Changes merged in #27

**Pull request closed without merging**